src/types/inc/sgrStack.hpp

```diff
@@ -14,8 +14,8 @@ Module Name:
 #pragma once
 
 #include "..\..\buffer\out\TextAttribute.hpp"
+#include "..\..\inc\til\enumset.h"

```

Comment:
Oh and I think this should be `#include <til/enumset.h>` for consistency...

Reply:
Cool. I didn't realise that would work. I'll do that now.